packages/thriver-masthead/lib/masthead.js

```diff
@@ -31,30 +31,40 @@ Template.masthead.onRendered(() => {
   const transitionSlider = () => {
     const activeItem = document.querySelector('.masthead menu.tabs > li a[aria-expanded="true"]');
     const menuItems = document.querySelectorAll('.masthead menu.tabs > li');
-
     // If there is no active item, make the first active
     if (!activeItem) {
-      document.querySelector('.masthead menu.tabs > li a')
-        .setAttribute('aria-expanded', true);
-    }
+      document.querySelector('.masthead menu.tabs > li a').setAttribute('aria-expanded', true);
+      document.querySelector('.masthead div.tabs > article').setAttribute('aria-hidden', false);
+    } else {
+      // Set the appropriate item as active
+      for (let i = 0; i < menuItems.length; i += 1) {
+        const item = menuItems[i];
 
-    // Set the appropriate item as active
-    for (let i = 0; i < menuItems.length; i += 1) {
-      const item = menuItems[i];
+        // Look for which item is currently active
+        if (item.children[0].getAttribute('aria-expanded') === 'true') {
+          // If this is the last element
+          if ((i + 1) === menuItems.length) menuItems[0].children[0].click();
+          else menuItems[i + 1].children[0].click();
 
-      // Look for which item is currently active
-      if (item.children[0].getAttribute('aria-expanded') === 'true') {
-        // If this is the last element
-        if ((i + 1) === menuItems.length) menuItems[0].children[0].click();
-        else menuItems[i + 1].children[0].click();
-
-        break;
+          break;
+        }
       }
     }
   };
 
-  setInterval(transitionSlider, 15000);
-
   // Start
   transitionSlider();
+
+  // Run Transition Interval if isPaused != true
+  const masthead = document.getElementById('masthead');
+  let isPaused = false;
+  const interval = window.setInterval(function() {
+    if (!isPaused) {
+      transitionSlider();
+    }
+  }, 5000);
+
+  // Toggle Pause
+  masthead.onmouseover = function () { isPaused = true; };
+  masthead.onmouseout = function () { isPaused = false; };
```

Comment:
We should never use the `oneventname` attributes, instead opting for `addEventListener`:

`masthead.addEventListener('mouseout', () => isPaused = false);`